metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DepublishRecordIdService.java

```diff
@@ -203,9 +203,6 @@ public WorkflowExecution createDepublishWorkflowExecution(MetisUser metisUser, S
     depublishPluginMetadata.setDatasetDepublish(datasetDepublish);
     workflow.setMetisPluginsMetadata(Collections.singletonList(depublishPluginMetadata));
 
-    // Validate the new workflow.
-    workflowUtils.validateWorkflowPlugins(workflow, null);
-
```

Comment:
This change now made the workflowUtils variable in this class unused. Best to remove it, right?

Reply:
Indeed, I missed that